shoop/core/locale/zh_Hans/LC_MESSAGES/django.po

```diff
@@ -747,6 +747,11 @@ msgstr "主分类"
 msgid "default price"
 msgstr "默认价格"
 
+#, fuzzy
+#| msgid "minimum purchase"
+msgid "minimum price"
+msgstr "最低购买"
```

Comment:
Gotta tune the params of shoop_makemessages to not include them by default. We already have many of them. Luckily those won't be included to the compiled MO files anyway, so these don't really matter.